lib/broadside/ecs/ecs_deploy.rb

```diff
@@ -11,18 +9,18 @@ class EcsDeploy < Deploy
       memory: 1000
     }
 
-    def initialize(target, opts = {})
-      super
+    def initialize(target_name, opts = {})
+      super(target_name, opts)
       Broadside.config.ecs.verify(:poll_frequency)
     end
 
     def deploy
       super do
-        unless EcsManager.service_exists?(@target.cluster, family)
-          raise Error, "No service for '#{family}'! Please bootstrap or manually configure one."
+        unless EcsManager.service_exists?(@target.cluster, @target.family)
+          raise Error, "No service for '#{@target.family}'! Please bootstrap or manually configure one."
```

Comment:
you know you use `@target.family` so much i think you should just use `delegate` and not change all these lines.  that's what the whole composition over inheritance thing is about.

```ruby
require 'active_support/core_ext/module/delegation'

attr_reader :target
delegate :family, to: :target
```
something like that should do it and save you a lot of typing